examples/mqtt/samplePropMqtt.conf

```diff
@@ -0,0 +1,35 @@
+#### [DEFAULT] ####
+
+EdgeSystemName = "EdgeSystem-Name"
+DeviceName = "Device-Name"
+MetricName = "Metric-Name"
+MetricWithOwnTsName = "Metric-With-Own-Ts-Name"
+BulkCollectedMetricName = "Bulk-Collected-Metric-Name"
+
+SystemPropList = {"Country":"USA-G", "State":"California", "City":"Palo Alto", "Location":"VMware HQ", "Building":"Promontory H Lab", "Floor":"First Floor"}
+DevicePropList = {"Country":"USA-G", "State":"California", "City":"Palo Alto", "Location":"VMware HQ", "Building":"Promontory H Lab", "Floor":"First Floor"}
+
+#### [IOTCC] ####
+
+WebSocketUrl = "Websocket-address-url"
+IotCCUID = "Username"
+IotCCPassword = "Password"
+
+#### [MQTT] ####
+
+MqttChannel = "mqtt-channel"
+MqttSubChannel1 = "mqtt-subchannel"
+BrokerIP = "mqtt-broker"
+BrokerPort = "mqtt-broker-port"
+keepalive = 60
+cacert = "/etc/broker-name/ca_certificates/ca.crt"
+certfile = "/etc/certs/client.crt"
+keyfile = "/etc/certs/client.key"
+mqtt_username = ""
```

Comment:
Please try to add example using username and password based mqtt approach as well.